classes/MusicAlbum/genre.rb

```diff
@@ -0,0 +1,20 @@
+class Genre
+  attr_reader :id, :name, :items
+
+  def initialize(name)
+    @id = generate_id
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
```

Comment:
Let's add this condition to not repeat the adding of the same item.

@items << item unless @items.include?(item)